astropy_helpers

```diff
@@ -1 +1 @@
-Subproject commit fedd7c52ba80e2246f83b8d1752baa3a153a33e2
+Subproject commit 160b48587680e11f12b66a51641a2601b869d1fa
```

Comment:
You will have to find which commit you did add this file and remove it.
Since this happened in the last commit you should be able to do:
```shell
$ git checkout xrt-1
$ git reset HEAD~1
$ git status # this should show you what has changed but need to be staged and what's not tracked
$ git add path/to/xrt.py path/to/test_xrt.py
$ git status # should show these two files as "staged" and astropy_helpers as modified but not staged
$ git commit -m 'pylint fixes'
$ git push --force origin xrt-1
```

If this has been done in an older commit then you should play with some extra branch to cherry-pick all that's new, unless it was included in a single commit by itself, then you could remove that commit.